app/views/downloads/_download_preface.html.erb

```diff
@@ -65,15 +65,24 @@
       <% end %>
     <% end %>
 
+    <%# i18n-tasks-use t("downloads.download_preface.translated_to.restricted_html") %>
+    <%# i18n-tasks-use t("downloads.download_preface.translated_to.revealed_html") %>
+    <%# i18n-tasks-use t("downloads.download_preface.translated_to.unrevealed_html") %>
```

Comment:
Any particular reason these hints are within the template instead of [in the helper](https://github.com/otwcode/otwarchive/blob/d87652399df73fa5e3b829b0089529501a381538/app/helpers/works_helper.rb#L115) where they're used?

Reply:
I had to think about it, but I think the reasoning is that the calls here "cause" the translation key to be used (i.e. without calling the helper here, the translation keys would be unused), and in contrast, the helper just "facilitates" the use